admin-js/tests/setupTests.js

```diff
@@ -39,7 +39,7 @@ beforeAll(async() => {
         return;
 
     if (global.__coverage__)
-        pythonProcess = spawn("coverage", ["run", global.pythonProcessPath], {"cwd": ".."});
+        pythonProcess = spawn("coverage", ["run", "--source=examples/", global.pythonProcessPath], {"cwd": ".."});
```

Comment:
I recommend putting such configuration into `.coveragerc` so that when different tools (like `pytest-cov` or some editor) run coverage differently, the same configuration is used.

Reply:
First, I'm trying to avoid including coverage of examples/ in the unit tests, so this is opting into it here in the JS integration test.
Second, I tried moving the `--cov` arguments from pytest.ini into .coveragerc, but codecov doesn't see the results in the same way for some bizarre reason. Result is that codecov doesn't report coverage for `__init__.py` files when using source in .coveragerc.

Reply:
Weird. That shouldn't be happening. Still, I'd check the coverage invocation separately from codecov to make sure it's actually measured. Note that the modern coverage.py versions use `source_pkgs` to point to the proper source. Ah and you may also require path mappings in the `coveragerc` file so that coveragepy knows that metrics recorded for the copy installed in `site-packages` should apply to the directory structure.

Reply:
And, perhaps, you need `relative_files = true` too.

Reply:
> avoid including coverage of examples/

There's a setting called `omit` that would do this.

Reply:
Yeah, there's actually several codecov issues reporting the same problem, so I've just left it as is. There doesn't seem to be a benefit to changing it at this point anyway.

> There's a setting called `omit` that would do this.

I don't think it makes much difference if we include in the CLI here or exclude in the CLI for pytest. The examples won't be touched by any pytest runs, only as part of the integration test. If we move everything into .coveragerc, then we'd probably want an omit here for tests/ anyway, so we'd only increase the number of CLI arguments needed.